mods/doors/init.lua

```diff
@@ -2,9 +2,8 @@
 doors = {}
 
 -- private data
-local _doors = {}
-_doors.registered_doors = {}
-_doors.registered_trapdoors = {}
+doors.registered_doors = {}
+doors.registered_trapdoors = {}
```

Comment:
These are now public. Please document them.

Reply:
Where should i list them, i cant find any other places where tables are listed/defined?

Reply:
Here's an example of another table with registered items:
https://github.com/minetest/minetest_game/blob/1b10a98e69538b061bacc243759a951ad2ec0ddc/game_api.txt#L319-L320
https://github.com/minetest/minetest_game/blob/1b10a98e69538b061bacc243759a951ad2ec0ddc/game_api.txt#L337-L346
However, latter part is already done (`Door definition`, `Trapdoor definition`)

Reply:
thanks il get it updated now.